.github/workflows/main.yml

```
@@ -2,139 +2,89 @@ name: main
-    - uses: actions/checkout@v4
-      with:
-        submodules: recursive
-    - name: build wheels
-      uses: pypa/cibuildwheel@v2.23.2
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v4.1.0
       env:
         CIBW_ARCHS: ${{ matrix.arch }}
-        CIBW_BUILD: ${{ matrix.cw_build }}
-        CIBW_TEST_SKIP: "*-win_arm64"
+        CIBW_BUILD: ${{ matrix.build }}
```

actionlint (1.7.12)

[error] object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type array<string>

(expression)